crates/store/re_log_types/src/path/entity_path_filter.rs

```diff
@@ -422,7 +429,20 @@ impl EntityPathFilter {
                     *effect,
                 )
             })
+            .inspect(|(ResolvedEntityPathRule { resolved_path, .. }, _)| {
+                if resolved_path.starts_with(&EntityPath::partition_properties()) {
+                    seen_properties = true;
+                }
+            })
             .collect();
+
+        if !seen_properties {
+            rules.insert(
+                ResolvedEntityPathRule::including_subtree(&EntityPath::partition_properties()),
+                RuleEffect::Exclude,
+            );
+        }
+
```

Comment:
so `resolve_strict`doesn't down that? Are we ok with that fact (when do we even use `resolve_strict`)? If so, then we should document that

Reply:
`resolve_strict` seems to be unused. Should I remove it?

Reply:
I've adapted it for now.

Reply:
would have removed it 🤷